test/test.sh

```diff
@@ -768,6 +770,22 @@ test_enable_disable() {
 	return 0
 }
 
+test_database_kill() {
```

Comment:
This test ensures that `PgUser*` still globally exists even if a database (which contains one of its passwords) is freed.